packages/braid-design-system/src/lib/components/Accordion/Accordion.docs.tsx

```diff
@@ -220,6 +220,36 @@ const docs: ComponentDocs = {
           </>,
         ),
     },
+    {
```

Comment:
Have you considered adding this as a util function to DRY it up?

```suggestion
    dataAttributeDocs({code: `
        <Accordion
          data={{ testid: 'accordion-1' }}
          // => data-testid="accordion-1"
        >
          <AccordionItem
            data={{ testid: 'accordion-item-1' }}
            // => data-testid="accordion-item-1"
          >
            ...
          </AccordionItem>
        </Accordion>
      `})
```

Or just the description section?

Reply:
Done!